client/src/components/Profile/EditProfile.tsx

```diff
@@ -0,0 +1,184 @@
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Formik, FormikHelpers } from 'formik';
+import useStyles from './useStyles';
+import { Select, MenuItem, InputLabel, Grid, FormGroup } from '@mui/material';
+
+const EditProfile = (): JSX.Element => {
+  const classes = useStyles();
+  const fillDate = [
+    1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31,
+  ];
+  const fillMonth = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const fillYear = [];
+  for (let i = 1910; i < 2022; i++) {
+    fillYear.push(i);
+  }
+  return (
+    <>
+      <Box
+        p={1}
+        style={{ justifyContent: 'center', fontSize: '20px', marginTop: '40px' }}
+        display="flex"
+        justifyContent="flex-end"
+        alignSelf="flex-end"
+        marginRight={5}
+      >
+        <InputLabel className={classes.labelEdit}>Edit Profile</InputLabel>
+      </Box>
+      <form className={classes.form} noValidate>
+        <Grid container component="main" className={classes.root}>
+          <Grid item xs={3} sm={4} md={3} className={classes.gridItem}>
+            <Box display="flex" justifyContent="center" flexDirection="column">
+              <Typography className={classes.label}>first name</Typography>
```

Comment:
I think it would be more appropriate to have the labels along with each text field as opposed to having them bunched up here